src/native/corehost/apphost/static/CMakeLists.txt

```diff
@@ -46,7 +46,12 @@ set(HEADERS
     ../../fxr_resolver.h
 )
 
-add_compile_definitions(NATIVE_LIBS_EMBEDDED)
+# TODO: Android uses a separate System.Security.Cryptography lib that does not have 
+# the exported functions the singlefilehost expects. 
+# Figure out how that's going to work and turn this back on.
```

Comment:
please file an issue for this, though I'm kinda surprised that the host depends on specific S.C.Cryptography functions?

Reply:
https://github.com/dotnet/runtime/issues/111491